### ISSUE TITLE
Does not support Arabic subtitles

### ISSUE THREAD

**Issue by Haikel-G** [paulpacifico/shutter-encoder#245]
Hey,

Is there any plan to make Shutter Encoder upload srt/ass arabic subtitles ?

**Comment by paulpacifico** [comment]
Hello,

That's should works with Arabic subtitles, which error do you get?

Paul.

**Comment by Haikel-G** [comment]
> Hello,
> 
> That's should works with Arabic subtitles, which error do you get?
> 
> Paul.

As you can see here, there's some errors in the display of the arabic characters with most fonts. Besides, it doesn't support RTL text direction 


![image](https://github.com/user-attachments/assets/ba7bdd0a-6d70-4229-ac1f-4b32f66f456b)

![image](https://github.com/user-attachments/assets/36bbd1da-4a6e-4c06-ba0b-b1dfb6567875)

**Comment by paulpacifico** [comment]
I suppose you've tried with different Arabic compatible fonts?

**Comment by Haikel-G** [comment]
Yes, by default it's Arial. I changed it to see if it works with different font, the problem is the same.

**Comment by paulpacifico** [comment]
Could you send me your subtitles for checking?

**Comment by Haikel-G** [comment]
> Could you send me your subtitles for checking?

Here's two lines from the original subtitle

[](https://uploadnow.io/f/RYyDJK8)

**Comment by chenlung** [comment]
The best thing would be to attach a short, reproducible subtitle file compressed to a .zip.

**Comment by Haikel-G** [comment]
[sub ar for test.zip](https://github.com/user-attachments/files/17818381/sub.ar.for.test.zip)

**Comment by paulpacifico** [comment]
Thanks, the bug is only visible on Windows, it seems to work properly using the '**Calibri**' font.

Paul.

**Comment by Haikel-G** [comment]
> Thanks, the bug is only visible on Windows, it seems to work properly using the '**Calibri**' font.
> 
> Paul.

what about the RTL text direction ?
